package.json

```diff
@@ -25,71 +25,88 @@
+    "build:declarations": "cpy --cwd types --parents '**/*' ../declarations && tsc -p tsconfig.declarations.json",
+    "build:code": "rollup --config rollup.build.js",
     "prepublishOnly": "npm run build",
-    "build": "npm run types:network && npm run js:network && npm run images:network",
+    "build": "npm run build:declarations && npm run build:code && npm run build:legacy:types && npm run build:legacy:code && npm run build:legacy:images",
     "watch": "npm run images:network && rollup --watch --config rollup.config.js",
     "watch-dev": "npm run watch-dev",
     "lint": "npm run lint:js && npm run lint:ts",
     "lint:js": "eslint '{lib,test}/**/*.js'",
     "lint:ts": "eslint '{lib,test}/**/*.ts'",
-    "clean": "rimraf 'dist/*'",
+    "clean": "rimraf \"dist\" \"declarations\" \"peer\" \"standalone\" \"vis-network*\"",
     "postinstall": "opencollective postinstall || exit 0"
   },
-  "dependencies": {
-    "@egjs/hammerjs": "2.0.15",
-    "component-emitter": "1.3.0",
+  "peerDependencies": {
     "keycharm": "0.2.0",
     "moment": "2.24.0",
-    "timsort": "0.3.0",
     "vis-data": "6.1.1",
     "vis-util": "1.1.6"
   },
   "devDependencies": {
     "@babel/core": "^7.5.5",
     "@babel/plugin-proposal-class-properties": "^7.5.5",
-    "@babel/plugin-transform-runtime": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-typescript": "^7.3.3",
     "@babel/register": "^7.5.5",
     "@babel/runtime": "^7.5.5",
+    "@egjs/hammerjs": "^2.0.15",
     "@types/chai": "^4.1.7",
     "@types/hammerjs": "^2.0.36",
     "@types/mocha": "^5.2.7",
     "@types/sinon": "^7.0.13",
     "@types/timsort": "^0.3.0",
     "@typescript-eslint/eslint-plugin": "^2.1.0",
     "@typescript-eslint/parser": "^2.1.0",
+    "@wessberg/rollup-plugin-ts": "^1.1.64",
     "babel-plugin-css-modules-transform": "^1.6.2",
     "babel-plugin-istanbul": "^5.2.0",
+    "babel-preset-minify": "^0.5.1",
     "chai": "^4.2.0",
+    "component-emitter": "^1.3.0",
     "cpy-cli": "^2.0.0",
     "eslint": "^6.3.0",
     "eslint-config-prettier": "^6.1.0",
     "eslint-plugin-prettier": "^3.1.0",
     "jsdoc": "^3.6.3",
     "jsdom": "15.1.1",
     "jsdom-global": "^3.0.2",
+    "keycharm": "^0.2.0",
     "mocha": "^6.2.0",
+    "moment": "^2.24.0",
     "nyc": "^14.1.1",
     "opencollective": "^1.0.3",
+    "postcss-assets": "^5.0.0",
     "prettier": "^1.18.2",
     "rimraf": "^3.0.0",
     "rollup": "^1.17.0",
     "rollup-plugin-babel": "^4.3.3",
-    "rollup-plugin-babel-minify": "^9.0.0",
     "rollup-plugin-commonjs": "^10.0.1",
+    "rollup-plugin-copy": "^3.1.0",
     "rollup-plugin-css-porter": "^1.0.2",
     "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-postcss": "^2.0.3",
+    "rollup-plugin-terser": "^5.1.2",
     "rollup-plugin-typescript2": "^0.24.0",
     "sinon": "^7.3.2",
-    "typescript": "^3.5.3"
+    "timsort": "^0.3.0",
+    "typescript": "^3.5.3",
+    "vis-data": "^6.1.1",
```

Comment:
How are vis-data and vis-util a devDependency?

Reply:
It's just a convenience thing. When you run `npm ci` it will automatically install them so that everything works. Technically speaking they're not dev deps but then everybody would have to install them manually for development and maintain their versions. Also the CI/CD as it is at the moment wouldn't install them and simply fail.